docs/contents/components/navigation/pagination/accessibility.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Pagination
+description: "`Pagination` is a component for managing the pagination and navigation of content."
+package_name: "@yamada-ui/pagination"
+is_tabs: true
```

Comment:
Please remove `is_tabs`.

Reply:
Thank for your follow. I had removed it.